src/Engine/Util/Util.cpp

```diff
@@ -0,0 +1,4 @@
+#include "Util.hpp"
+
+constexpr double Util::PI;
```

Comment:
what is this for? this isn't how constexpr works

Reply:
Without this file, compiler gives me the following:

- undefined reference to 'Util::infinity'

The same error happened with the constexpr in CollisionSystem.cpp.

Solution to problem was found here: https://stackoverflow.com/questions/8016780/undefined-reference-to-static-constexpr-char

More detail:

src/Engine/Component/CollisionComponents/CollisionComponent.cpp.o: In function `(anonymous namespace)::detMeshSpan(int, glm::tvec3<float, (glm::precision)0> const*)':
CollisionComponent.cpp:(.text+0x1025): undefined reference to `Util::infinity'
collect2: error: ld returned 1 exit status

Reply:
right, i forgot constexpr wasn't super fleshed out in c++11. What constexpr is supposed to mean is inline and compile-time. Oh well